bundles/specmate-dummy-data/src/com/specmate/dummydata/DummyDataService.java

```diff
@@ -87,7 +88,12 @@ private void fillDummyData() throws SpecmateException {
 			transaction.getResource().getContents().add(testFolder);
 
 			try {
-				transaction.commit();
+				transaction.doAndCommit(new IChange<Object>() {
+					@Override
+					public Object doChange() throws SpecmateException {
+						return null;

```

Comment:
same here, the transaction.getResource.... line should go inside the method call